rma/views/rma_tag_views.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="rma_tag_view_search" model="ir.ui.view">
+        <field name="model">rma.tag</field>
+        <field name="arch" type="xml">
+            <search string="RMA Tags">
+                <field name="name" />
+                <filter
+                    string="Archived"
+                    name="inactive"
+                    domain="[('active','=',False)]"
+                />
+                <filter
+                    string="Active"
+                    name="active"
+                    domain="[('active','!=',False)]"
+                />
+            </search>
+        </field>
+    </record>
+    <record id="view_rma_tag_form" model="ir.ui.view">
+        <field name="name">Rma Tags</field>
+        <field name="model">rma.tag</field>
+        <field name="arch" type="xml">
+            <form string="RMA Tag">
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            class="oe_stat_button"
+                            type="object"
+                            name="toggle_active"
+                            icon="fa-archive"
+                        >
+                            <field
+                                name="active"
+                                widget="boolean_button"
+                                options='{"terminology": "archive"}'
+                            />
+                        </button>
+                    </div>
```

Comment:
```suggestion
```
Remove this, the archive/unarchive actions are available without doing this. 
(Optional) If you want you can add a ribbon: 
```XML
<field name="active" invisible="1" />
<widget name="web_ribbon" text="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
```